docs/src/extend/custom-processors.md

```diff
@@ -140,7 +140,56 @@ By default, ESLint does not perform autofixes when a custom processor is used, e
 
 You can have both rules and custom processors in a single plugin. You can also have multiple processors in one plugin. To support multiple extensions, add each one to the `processors` element and point them to the same object.
 
-**The `meta` object** helps ESLint cache the processor and provide more friendly debug message. The `meta.name` property should match the processor name and the `meta.version` property should match the npm package version for your processors. The easiest way to accomplish this is by reading this information from your `package.json`.
+### How `meta` Objects are Used
+
+The `meta` object helps ESLint cache configurations that use a processor and to provide more friendly debug messages. The `meta.name` property should match the processor name and the `meta.version` property should match the npm package version for your processors. The easiest way to accomplish this is by reading this information from your `package.json`.
+
+#### Plugin `meta` Object
+
+The [plugin `meta` object](plugins#meta-data-in-plugins) provides information about the plugin itself. When a processor is specified using the string format `plugin-name/processor-name`, ESLint automatically uses the plugin `meta` to generate a name for the processor. This is the most common case for processors.
+
+Example:
+
+```js
+// eslint.config.js
+import example from "eslint-plugin-example";
+
+export default [
+    {
+        plugins: {
+            example
+        },
+        processor: "example/processor-name"
+    },
+    // ... other configs
+];
+```
+
+In this example, the processor name is `"example/processor-name"`, and that's the value that will be used for serializing configurations.
```

Comment:
Any idea why GitHub marks this dot and a few other dots below in red?

![image](https://github.com/user-attachments/assets/395fd35c-2f3d-4d0c-ba83-0469ffccde50)

Reply:
It seems like after the code block on line 128, the syntax highlighting gets weird. Maybe it doesn't know how to deal with intended code blocks?

Reply:
Good catch, yeah that seems to be the case. I thought it might be marking a dot-like character that isn't the common dot or perhaps some accidentally inserted unusual whitespace around it, but all characters are just common characters and this indeed seems to be a syntax highlighting issue.